store/kv/kv.go

```diff
@@ -102,6 +103,22 @@ func measureLow(ctx context.Context, name string, f func() error) {
 		metrics.FdbRequests.Tagged(tags).Histogram("histogram", tally.DefaultBuckets)
 		defer metrics.FdbRequests.Tagged(tags).Histogram("histogram", tally.DefaultBuckets).Start().Stop()
 	}
+	if config.DefaultConfig.DatadogTrace.Enabled {
+		spanOpts := []tracer.StartSpanOption{
+			tracer.ServiceName("tigris.fdb.kv"),
+			tracer.ResourceName(name),
```

Comment:
I would actually move the resource name mangling logic into that helper as well, if applicable, from `server/"middddddddddddleware"/ddtrace.go`. :- )

Reply:
Done.